.golangci.yml

```diff
@@ -16,6 +16,12 @@ issues:
       linters:
         - dupl
         - lll
+    - path: "pkg/monitoring/prometheusagent/secret.go"
+      linters:
+        - gosec
+    - path: "pkg/monitoring/mimir/service.go"
+      linters:
+        - gosec
```

Comment:
What nasty things are we doing there which requires to ignore gosec ?

Reply:
gosec was complaining about const names such as `ingressAuthSecretName`

Reply:
Then please fix the warning rather than ignoring the whole file. We want to keep `gosec` checking for potential security issues.

Reply:
We can't really fix them all because if a variable name contains secret or api key it will complain but we Can use a single Line comment over excluding a full file

Reply:
Like this : https://github.com/giantswarm/observability-operator/blob/ec1abdec2ce3c7a4a95c2faeb7cd431f25dec52d/main.go#L69

Reply:
Oh nice ! I'm gonna do that then :)